apps/participants/views.py

```diff
@@ -114,6 +114,15 @@ def invite_participant_to_team(request, pk):
         response_data = {'error': 'User does not exist with this email address!'}
         return Response(response_data, status=status.HTTP_406_NOT_ACCEPTABLE)
 
+    if not is_user_part_of_participant_team(request.user, participant_team):
```

Comment:
This cannot be triggered using the UI, yes?

Reply:
I didn't get your question here.